src/main/java/frc/robot/OperatorInterface.java

```diff
@@ -20,89 +17,82 @@
-                .whenPressed(commandFactory.getStartShooterCommandGroup())
-                .whenReleased(commandFactory.getStopShooterCommandGroup())
+                .whenPressed(commandFactory.startShooter())
+                .whenReleased(commandFactory.stopShooter())
                 .add();
         
-        operatorPanelManager.addButton(RobotConstants.BUTTONS.ENABLE_AUTO_HOOD)
-                .whenPressed(subsystemManager.getShooterSubsystem().enableAutoShooting())
-                .whenReleased(subsystemManager.getShooterSubsystem().disableAutoShooting())
-                .add();
+        //operatorPanelManager.addButton(RobotConstants.BUTTONS.ENABLE_AUTO_HOOD)
+        //        .whenPressed(subsystemManager.getShooterSubsystem().enableAutoShooting())
+        //        .whenReleased(subsystemManager.getShooterSubsystem().disableAutoShooting())
+        //        .add();
         
         operatorPanelManager.addButton(RobotConstants.BUTTONS.DEPLOY_INTAKE)
-                .whileHeld(commandFactory.getStartIntakeCommandGroup())
-                .whenReleased(commandFactory.getStopIntakeCommandGroup())
+                .whileHeld(commandFactory.startIntake())
+                .whenReleased(commandFactory.stopIntake())
                 .add();
         
         operatorPanelManager.addButton(RobotConstants.BUTTONS.HOOD_FORWARD_ADJUST)
-                .whileHeld(new AdjustRaiseHoodCommand(subsystemManager.getShooterSubsystem()))
+                .whileHeld(commandFactory.nudgeHoodForward())
                 .add();
         
         operatorPanelManager.addButton(RobotConstants.BUTTONS.HOOD_BACKWARD_ADJUST)
-                .whileHeld(new AdjustBackwardHoodCommand(subsystemManager.getShooterSubsystem()))
+                .whileHeld(commandFactory.nudgeHoodBackward())
                 .add();
                 
-        operatorPanelManager.addButton(RobotConstants.BUTTONS.SELECT_PRESET_1)
-                .whenPressed(subsystemManager.getShooterSubsystem().selectPreset1())
-                .add();
+        //operatorPanelManager.addButton(RobotConstants.BUTTONS.SELECT_PRESET_1)
+        //        .whenPressed(subsystemManager.getShooterSubsystem().selectPreset1())
+        //        .add();
         
-        operatorPanelManager.addButton(RobotConstants.BUTTONS.SELECT_PRESET_2)
-                .whenPressed(subsystemManager.getShooterSubsystem().selectPreset2())
-                .add();
+        //operatorPanelManager.addButton(RobotConstants.BUTTONS.SELECT_PRESET_2)
+        //        .whenPressed(subsystemManager.getShooterSubsystem().selectPreset2())
+        //        .add();
         
         drive = subsystemManager.getDriveSubsystem();
         
         joystickManager.addButton(RobotConstants.BUTTONS.SNAP_TO_TARGET)
-                .whenPressed(new SnapToVisionTargetCommand(drive).withTimeout(5))
+                .whenPressed(commandFactory.snapToVisionTargetCommand().withTimeout(5))
```

Comment:
I know I also did this but just something I noticed was the fact that some commands have timeouts while others do not. Is there a way that we can make every command have a timeout. I know that decorators aren't meant for this but still. Also magic number.

Reply:
yeah i think we just need to add withTimeout() to them all, or extend our base class.

I like the idea of having it in our base class slightly better, but not enough better to re-invent an entire catalog of functionality available to us if we dont. EG, you cant use WaitCommand, WaitUntilCommand, PrintCommand, SequentialCommand, etc while also using ours.